### PR TITLE
Improved code style.

### PR DESCRIPTION
**Pull request by AnikHasibul**
This change is [Reviewable](https://reviewable.io/reviews/dgraph-io/badger/754)

### CODE REVIEW

**Comment by manishrjain** [comment]
Thanks for your changes. Can you please sign the agreement before we accept this PR?

**Comment by manishrjain** [comment]
Ah... looks like you signed already. Only the bot didn't. Should be okay to merge.

**Comment by AnikHasibul** [comment]
> Should be okay to merge. 
Yeah! :1st_place_medal:

File: db.go [hunk 1]
```diff
@@ -21,6 +21,7 @@ import (
 	"encoding/binary"
```
Comment (mangalaman93) [inline db.go]: This doesn't look gofmt formatted, please check!

Comment (AnikHasibul) [reply db.go]: Formatted :checkered_flag:

**Comment by mangalaman93** [comment]
LGTM
@ashish-goswami please merge after having a look once.

**Comment by mangalaman93** [comment]
@ashish-goswami check why CI builds are failing for this PR

**Comment by AnikHasibul** [comment]
Thank you so much @mangalaman93

**Comment by mangalaman93** [comment]
@AnikHasibul we can't accept changes from ImgBot unless CLA can be signed. Please remove the commit from this PR.

**Comment by AnikHasibul** [comment]
Just reverted the commit came from @ImgBotApp . I think it's okay now!

**Comment by mangalaman93** [comment]
Well, reverting the commit is not enough. You have to completely remove the commit from the history as well.

**Comment by manishrjain** [comment]
I force merged it as an admin to avoid the dep on @ImgBotApp .

**Pull request merged**